pom.xml

```diff
@@ -443,6 +444,56 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
in checkstyle, there is one rule to check asf header. If team decide to use apache rat plugin, we may need remove checkstyle rule. 
Related files:
+ https://github.com/apache/incubator-dubbo/blob/master/codestyle/checkstyle-header.txt
+ https://github.com/apache/incubator-dubbo/blob/master/codestyle/checkstyle.xml#L10

Reply:
ok